features/step_definitions/landing_page_steps.rb

```diff
@@ -0,0 +1,20 @@
+When(/^I visit the LandingPage$/) do
+  landing_page.load
+end
+
+Then(/^I want to be able to input my age$/) do
+  expect(page).to have_css('#your_details_form_age')
+end
+
+And(/^I want to be able to input my gender$/) do
+  expect(page).to have_css('#your_details_form_gender')
+end
+
+And(/^I want to be able to input my salary$/) do
+  expect(page).to have_css('#your_details_form_salary')
```

Comment:
This will allow you to re-write as: `expect(@landing_page).to have_salary_field`. The original intent of the code is not to check for the contents of the _salary_field_ so this is the closest you can get to refactoring the original piece of code `expect(page).to have_css('#your_details_form_salary')`.